lib/pluginmanager/update.rb

```diff
@@ -51,7 +52,9 @@ def update_gems!
 
     # any errors will be logged to $stderr by invoke!
     # Bundler cannot update and clean gems in one operation so we have to call the CLI twice.
-    output = LogStash::Bundler.invoke!(:update => plugins)
+    options         = {:update => plugins}
```

Comment:
I don't like aligned `=` but that's a personal taste. they always endup being unaligned anyway so it pointless